### PR TITLE
feat: [RGOeX-26759] adaptation for use on Redwood release

### PR DESCRIPTION
**Pull request by DmytroAlipov**
### Youtrack:
https://youtrack.raccoongang.com/issue/RGOeX-26759

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def package_data(pkg, roots):
         'requests>=2.9.1,<3.0.0',
         'babelfish>=0.5.5,<0.6.0',
         'XBlock>=1.4.0,<2',
-        'xblock-utils>=2.1,<4'
+        'xblock-utils==4.0.0'
```
Comment (DmytroAlipov) [inline setup.py:55]: This XBlock is deprecated. But the functionality is imported from the new Xblock and we can use it.
https://github1s.com/openedx-unsupported/xblock-utils/blob/v4.0.0/xblockutils/studio_editable.py

Comment (dyudyunov) [reply setup.py:55]: Why didn't you remove this dependency and replace its usage with the XBlock package (where the code was migrated)?

`'XBlock>=1.4.0,<2',` also conflicts with edx-platform's requirements where it is pinned on `xblock[django]==4.0.1` and this version [contains the migrated code](https://github.com/openedx/XBlock/tree/v4.0.1/xblock/utils) 

Please update your PR:
- remove deprecated xblock-utils
- replace its usage with the corresponding XBlock code 
- update the XBlock requirement corresponding to the edx-platform version
- re-check if there are any other outdated packages to update compared to edx-platform requirements

Comment (DmytroAlipov) [reply setup.py]: done

File: requirements.txt [hunk 2]
```diff
@@ -1,8 +1,7 @@
 pycaption==2.1.0
-xblock-utils>=2.1,<4
 requests>=2.9.1,<3.0.0
 babelfish>=0.5.5,<0.6.0
-XBlock==1.4.0,<2
-django-statici18n==1.9.0
+xblock[django]==4.0.1
+django-statici18n==2.5.0
```
Comment (dyudyunov) [inline requirements.txt:5]: ditto
`django-statici18n>=2.5.0,<3.0.0`

File: setup.py [hunk 3]
```diff
@@ -51,8 +51,7 @@ def package_data(pkg, roots):
         'pycaption==2.1.0',
         'requests>=2.9.1,<3.0.0',
         'babelfish>=0.5.5,<0.6.0',
-        'XBlock>=1.4.0,<2',
-        'xblock-utils>=2.1,<4'
+        'XBlock==4.0.1',
```
Comment (dyudyunov) [inline setup.py:54]: ditto

**Review by dyudyunov — CHANGES_REQUESTED** [review]

File: requirements.txt [hunk 4]
```diff
@@ -1,8 +1,7 @@
 pycaption==2.1.0
-xblock-utils>=2.1,<4
 requests>=2.9.1,<3.0.0
 babelfish>=0.5.5,<0.6.0
-XBlock==1.4.0,<2
-django-statici18n==1.9.0
+xblock[django]==4.0.1
```
[the remark this answers is not in the document]
Comment (DmytroAlipov) [reply requirements.txt]: You are probably right, it will be more correct.

File: requirements.txt (hunk 2 above)
Comment (DmytroAlipov) [reply requirements.txt]: done

File: setup.py (hunk 3 above)
Comment (DmytroAlipov) [reply setup.py]: done

**Review by dyudyunov — APPROVED** [review]

**Pull request merged**